Active_Record/Rakefile

```diff
@@ -0,0 +1,56 @@
+require './config/initializers/database'
+
+namespace :db do
+
+  desc "Create the database"
+  task :create do
+    ActiveRecord::Base.establish_connection(CONFIG_DATABASE.except('database'))
+    ActiveRecord::Base.connection.create_database(CONFIG_DATABASE["database"])
```

Comment:
use single paws for a non-mutable strings

Reply:
done